modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -61,6 +61,14 @@ def config
         base_path_template: '/services/appeals/contestable-issues/{version}',
         name: 'contestable_issues',
         tags: [{ name: 'Contestable Issues', description: '' }]
+      ),
```

Comment:
Adding another explicit doc path key to this dict introduced a new problem based on how rswag works - in short:

For now, when we run any of the rswag rake tasks, _all_ of the output docs listed here (contestable issues, legacy appeals, and decision reviews) will be generated at once, regardless of which task is running. Those that are specifically targeted by the task (for example `bundle exec rake rswag:appeals_api:la:prod` for the production legacy appeals docs) will be generated correctly, but all the other docs listed as keys here will be overwritten with blank content.

Why this happens:
* Rswag is intended to be used to generate all of the configured docs at once.
    * In [the example app in rswag's repo](https://github.com/rswag/rswag/tree/master/test-app), there is one single `swaggerize` rake task
    * This rake task runs _all_ rswag specs at once, and rswag determines which output files to update based on the `swagger_doc:` setting at the top of each spec file, [like here](https://github.com/rswag/rswag/blob/master/test-app/spec/integration/openapi3_spec.rb#L8). This way, all docs are generated correctly at the same time.
* Our setup is slightly different: each of our rake tasks targets a specific output file, and only the specs for that single API actually run - we do this by using the `PATTERN` environment variable [via these settings](https://github.com/department-of-veterans-affairs/vets-api/pull/12576/files#diff-3959fe998d856cb06d8cabe389bdee804f212d00616790f3717f9c050e86a0ddR17-R32).
* Nevertheless, rswag still wants to generate all of the docs configured in this file at once - so even though only one API's specs are running, JSON output is generated for all of the entries in this config. And since the `PATTERN` we set targets only specific spec files, the other files are blank.

If that all sounds too complicated, I agree... this problem will go away once we're able to explicitly list each doc here and get rid of the use of `PATTERN` and match the usage in the example app from the rswag repo.